modules/core/arrow-extras-extensions/src/main/kotlin/arrow/data/extensions/eithert.kt

```diff
@@ -8,8 +8,10 @@ import arrow.core.extensions.either.traverse.traverse
 import arrow.data.*
 import arrow.extension
 import arrow.typeclasses.*
+import arrow.undocumented
 
 @extension
+@undocumented
```

Comment:
You can use this annotation to tell the documentation compiler not to inline the documentation in this because.

This is done because we're currently missing some features to figure out how to currently to inline and adjust code snippets to be valid for monad transformers.